common/aicore/path_finding.cpp

```diff
@@ -3266,6 +3266,7 @@ pf_path_new_to_start_tile(const struct pf_parameter *param)
 void pf_path_destroy(struct pf_path *path)
 {
   if (nullptr != path) {
+    delete path->positions;
```

Comment:
This will be an alloc-dealloc mismatch when allocated with `new[]`. Change all allocations to `new[]` using `new[1]` when a single element is needed

Reply:
Fixed. I didnt (also 500turns with ASAN) didn't spot anything else.